.github/workflows/test_afl_fuzzer.yml

```diff
@@ -0,0 +1,30 @@
+name: Test AFL fuzzer host (mDNS)
```

Comment:
This only checks that the mdns fuzzer layers compile, correct? We shouldn't call it `AFL fuzzer`...